trade-finance-manager-api/src/v1/controllers/update-portal-deal-from-MIA-to-MIN.js

```diff
@@ -18,6 +18,9 @@ const updatePortalDealFromMIAtoMIN = async (dealId, dealType, checker) => {
       dealId,
       dealUpdate,
     );
+
+    // adds portal activity object for min submission and facilities changed -> issued
+    update = await api.updateGefMINActivity(dealId);
```

Comment:
Adding a test for the spy being called will be testing this bit of the code 😄

Reply:
done